zebra-network/src/peer_set/initialize/tests/vectors.rs

```diff
@@ -1245,7 +1245,7 @@ async fn remnant_nonces_from_outbound_connections_are_limited() {
     let hs = peer::Handshake::builder()
         .with_config(config.clone())
         .with_inbound_service(nil_inbound_service)
-        .with_user_agent(crate::constants::USER_AGENT.to_string())
+        .with_user_agent(("Test user agent").to_string())
```

Comment:
Why don't we go with the real user agent?

Reply:
We don't have a "real user agent" here if that means a Zebra user agent. Zebra user agent is built in the `zebrad` crate and passed to `zebra-network`. Then this user agent is used https://github.com/ZcashFoundation/zebra/blob/8615dd750f28cb2b6d1956e022f26e83067ec011/zebra-network/src/peer_set/initialize.rs#L135
Here the test is starting a new handshake so it does not have any user agent being pass, so that string was added.

Reply:
Ah, of course. I should have thought of that. Thanks for the clarification.